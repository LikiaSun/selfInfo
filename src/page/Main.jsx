import React from 'react';
import { Link } from 'react-router-dom';

const Main = () => {
  return (
    <div>
      <Link to="/profile">Home</Link>
      <h1>Likia</h1>
    </div>
  );
};

export default Main;
