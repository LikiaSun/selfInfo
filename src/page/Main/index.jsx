import React from 'react';

import Navbar from '@/components/Navbar';
import Header from '@/components/MainHeader';

const Main = () => {
  return (
    <main>
      <Navbar />
      <Header />
    </main>
  );
};

export default Main;
