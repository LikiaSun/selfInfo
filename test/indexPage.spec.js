import React from 'react';
import { shallow } from 'enzyme';
import Main from '../src/page/Main';

describe('Index page render', () => {
  it('should render page', () => {
    const page = shallow(<Main />);

    expect(page).toMatchSnapshot();
  });
});
