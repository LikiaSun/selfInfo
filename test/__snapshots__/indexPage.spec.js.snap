// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page render should render page 1`] = `
<div>
  <Link
    to="/profile"
  >
    Home
  </Link>
  <h1>
    Likia
  </h1>
</div>
`;
